cedar-policy-cli/README.md

```diff
@@ -10,6 +10,10 @@ Cedar.
 This app uses the annotation `@id("PID")` as a simple way to define policy ids.
 This usage is not standard and annotations have custom use depending on the app.
 
+## Install
+
+To install the CLI, run `cargo install cedar-policy-cli`.
```

Comment:
This command will also require `cargo`, probably from rustup. Should these instructions also mention Rustup (like the Build ones do below)?

Reply:
Thanks for your feedback! I also added a mention for Rustup!